resources/darwin/Credits.rtf

```diff
@@ -0,0 +1,11 @@
+{\rtf1\ansi\ansicpg1252\cocoartf1348\cocoasubrtf170
```

Comment:
Intentional?

Reply:
I think that's just what rtf files look like under the hood

Reply:
No, not intentional! Thanks!